src/PendingScopedFeatureInteraction.php

```diff
@@ -249,9 +249,12 @@ public function unless($feature, $whenInactive, $whenActive = null)
      */
     public function activate($feature, $value = true)
     {
-        Collection::wrap($feature)
+        $features = Collection::wrap($feature)
             ->crossJoin($this->scope())
-            ->each(fn ($bits) => $this->driver->set($bits[0], $bits[1], $value));
+            ->map(fn ($bits) => ['feature' => $bits[0], 'scope' => $bits[1], 'value' => $value])
+            ->all();
+
+        $this->driver->setAll($features);
```

Comment:
`$this->driver` is our decorator which implements the new contract, so calling `setAll` here is safe.